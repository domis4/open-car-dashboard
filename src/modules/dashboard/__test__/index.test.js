import React from 'react'
import { shallow } from 'enzyme'
import Dashboard from '../index'

describe('dashboard module index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<Dashboard />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
