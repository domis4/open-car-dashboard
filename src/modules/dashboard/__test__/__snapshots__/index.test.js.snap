// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard module index.js expects component to match snapshot 1`] = `
<div
  className="View-root-01"
>
  <WithStyles(Grid)
    container={true}
    spacing={24}
  >
    <WithStyles(Grid)
      item={true}
      xs={3}
    >
      <WithStyles(Paper)>
        <WithStyles(View)
          callback={[Function]}
          image={
            Object {
              "process": [Function],
            }
          }
          title="Music"
        />
      </WithStyles(Paper)>
    </WithStyles(Grid)>
    <WithStyles(Grid)
      item={true}
      xs={3}
    >
      <WithStyles(Paper)>
        <WithStyles(View)
          callback={[Function]}
          image={
            Object {
              "process": [Function],
            }
          }
          title="Settings"
        />
      </WithStyles(Paper)>
    </WithStyles(Grid)>
    <WithStyles(Grid)
      item={true}
      xs={3}
    >
      <WithStyles(Paper)>
        <WithStyles(View)
          callback={[Function]}
          image={
            Object {
              "process": [Function],
            }
          }
          title="Google Maps"
        />
      </WithStyles(Paper)>
    </WithStyles(Grid)>
    <WithStyles(Grid)
      item={true}
      xs={3}
    >
      <WithStyles(Paper)>
        <WithStyles(View) />
      </WithStyles(Paper)>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</div>
`;
